.github/workflows/bundlewatch.yml

```diff
@@ -2,8 +2,8 @@ name: Bundlewatch
 
 on:
   push:
-    branches-ignore:
-      - "dependabot/**"
```

Comment:
We tried something to remove dependabot from the CI (still actionnable by adding a tag on a dependabot branch). Added `    if: "!(github.actor == 'dependabot[bot]')"` at each job level. Could be an alternative here if you don't want the action mentionned above.